pkg/controller/core/workload_controller_test.go

```diff
@@ -389,17 +392,19 @@ func TestReconcile(t *testing.T) {
 					State: kueue.CheckStateReady,
 				}).
 				Condition(metav1.Condition{
-					Type:    "Admitted",
-					Status:  "True",
-					Reason:  "Admitted",
+					Type:    kueue.WorkloadAdmitted,
```

Comment:
It's ok to use strings in tests, no need to change it imo. It is more readable that way.

Reply:
Reverted